gradio/chat_interface.py

```diff
@@ -170,85 +173,84 @@ def __init__(
             if description:
                 Markdown(description)
 
```

Comment:
looks like a lot of change but just moved the chat buttons before the textbox and got rid of the parent panel column